lib/Linter.js

```diff
@@ -586,6 +589,35 @@ function stripUnicodeBOM(text) {
     return text;
 }
 
+/**
+ * Get the severity level of a rule (0 - none, 1 - warning, 2 - error)
+ * Returns 0 if the rule config is not valid (an Array or a number)
+ * @param {Array|number} ruleConfig rule configuration
+ * @returns {number} 0, 1, or 2, indicating rule severity
+ */
+function getRuleSeverity(ruleConfig) {
+    if (typeof ruleConfig === "number") {
```

Comment:
As you said, this logic is unchanged from the old version. So I'm hoping that means we've normalized to numbers by this point. I just wish we knew for sure and could indicate that in the comment. I certainly don't want to actually change too much logic here.